### PR TITLE
Adds a new flavor book.

### PR DESCRIPTION
**Pull request by bruhmogus**
Writing credits go to Nettingman, small edit made by me, FBI, changing all mentions of "reptilian" to "unathi" to fit with EE lore.
 # Description

Adds the new flavor book, "The Lusty Unathi Janitor" that was pr'd to wizden and is still open iirc

---

# TODO

- [x] Add book prototype
- [x] Add locale

---

# Changelog

:cl:
- add: Added the Lusty Unathi Janitor
- remove: Removed sanity.

### CODE REVIEW

**Comment by bruhmogus** [comment]
all test fails unrelated

File: Resources/Prototypes/Entities/Objects/Misc/books_author.yml [hunk 1]
```diff
@@ -657,3 +657,24 @@
   - type: Paper
     content: book-text-janitor
 
+- type: entity
+  parent: BookBase
+  id: BookUnathiJanitor
+  name: the lusty unathi janitor
+  description: A dirty book with even dirtier contents.
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Entities/Objects/Misc/books_author.yml:664]:
```suggestion
  description: A dirty book with even dirtier contents. You feel like you should burn it.
```

**Review by TheDoctor1977 — COMMENTED** [review]
the funny


do you have proper approval from the original author to make this port

**Comment by bruhmogus** [comment]
> the funny
> 
> do you have explicit and direct approval from the original author to make this port

direct approval acquired

**Comment by Erisfiregamer1** [comment]
i hope you know what you've done

**Comment by bruhmogus** [comment]
> i hope you know what you've done

i do. trust me.

**Pull request merged**